CHANGELOG.md

```diff
@@ -35,6 +35,9 @@
   Patch by Alex Waygood.
 - Speedup `isinstance(3, typing_extensions.SupportsIndex)` by >10x on Python
   <3.12. Patch by Alex Waygood.
+- Add `__orig_bases__` to non-generic TypedDicts, call-based TypedDicts, and
```

Comment:
Looks like you added this to the wrong PR :)

Reply:
Yep just swapped around